src/functions-and-arrays.js

```diff
@@ -1,19 +1,145 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 < num2) {
+    return num2;
+  } else if (num1 > num2) {
+    return num1;
+  } else {
+    return num1, num2;
```

Comment:
Is there a reason why a comma is used here? Wouldn't simply returning one of the two arguments do?